cql-tests-runner.js

```diff
@@ -86,6 +89,9 @@ class Result {
 async function main() {
     const args = process.argv.slice(2);
     let apiUrl = 'https://cloud.alphora.com/sandbox/r4/cds/fhir/$cql';
+    let cqlEngine = new CQLEngine(apiUrl);
+    cqlEngine.cqlVersion = '2.4.0';
```

Comment:
I think this is supposed to be setting the "engineVersion"? cqlVersion here should be the CQL specification version, 1.3, 1,4, or 1.5, yes?